packages/components/src/internal/components/domainproperties/list/ListDesignerPanels.tsx

```diff
@@ -214,6 +214,12 @@ export class ListDesignerPanelsImpl extends React.PureComponent<Props & Injected
             })
             .catch(response => {
                 const exception = resolveErrorMessage(response);
+
+                // Issue 49113: Better handling for case where the only error is a form error
+                if (!exception) {
+                    response = response.setIn(['domainException', 'exception'], resolveErrorMessage(response.domainException.exception));
```

Comment:
I assume `domainException` will always exist in this case, or does this want a question mark in there?

Reply:
It should exist, but I'll add the null check just in case.